src/components/ebay-breadcrumbs/index.marko

```diff
@@ -6,7 +6,7 @@ $ const {
     a11yHeadingTag,
     a11yMenuButtonText,
     class: inputClass,
-    items: attrTagItems,
+    item: attrTagItems = [],
     ...htmlInput
 } = input;
 
```

Comment:
GitHub won't let me comment there, but the line below this doesn't need `|| []` anymore because of the default above